WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -2011,8 +2011,18 @@
 			);
 			target = 0CED016F2D95B897003015CF /* Keystone */;
 		};
+		3F1A48232DA3B23900786B92 /* Exceptions for "WordPressData" folder in "Keystone" target */ = {
+			isa = PBXFileSystemSynchronizedBuildFileExceptionSet;
+			membershipExceptions = (
+				"Theme+Swift.swift",
+			);
+			target = 0CED016F2D95B897003015CF /* Keystone */;
+		};
```

Comment:
This and the rest of the diff in the project file come from me placing these new file in `Sources/WordPressData`, where it should live in the end, but needing to add it to Keystone, Jetpack, and WordPress while the framework is being worked out (see https://github.com/wordpress-mobile/WordPress-iOS/pull/24378)